tests/periph/fmc/Makefile

```
@@ -0,0 +1,11 @@
+BOARD ?= stm32f429i-disc1
+include ../Makefile.periph_common
+
+USEMODULE += benchmark
+USEMODULE += periph_fmc
```

```suggestion
FEATURES_REQUIRED += periph_fmc
```

I'm realizing that it's required here to limit the build to boards providing that feature (the sdram/nor_sram versions). Otherwise nothing will limit the build if the `FEATURES_OPTIONAL` only are kept.